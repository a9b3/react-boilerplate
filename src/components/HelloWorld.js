// @flow

import React  from 'react'

import styles from './HelloWorld.scss'

export default class HelloWorld extends React.PureComponent<{}> {
  render() {
    return <div className={styles.hello}>Hello World</div>
  }
}
