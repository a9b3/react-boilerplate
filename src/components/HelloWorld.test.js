// @flow

import React      from 'react'
import renderer   from 'react-test-renderer'

import HelloWorld from './HelloWorld.js'

test('Hello World', () => {
  const component = renderer.create(<HelloWorld />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
