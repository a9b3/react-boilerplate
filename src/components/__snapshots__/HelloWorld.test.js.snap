// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World 1`] = `
<div
  className="hello"
>
  Hello World
</div>
`;
